joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -84,32 +85,31 @@ JointTrajectoryController::update()
     return CONTROLLER_INTERFACE_RET_SUCCESS;
   }
 
-  // when no traj msg has been received yet
-  if (!traj_point_active_ptr_ || (*traj_point_active_ptr_)->is_empty()) {
-    return CONTROLLER_INTERFACE_RET_SUCCESS;
-  }
```

Comment:
So am I on the early returns. But I needed publish the state despite there being no current goal, and so the changes turned out that way.

Reply:
why not calling `publish_state()` before that logic then?

Reply:
Because i need the results of the sampled trajectory. In PR #26 action feedback will also be given and I'll need to rearrange this update function anyway.